### PR TITLE
Add recipe for pybiomart

### PR DESCRIPTION
**Pull request by xialiu71**
Add recipe for pybiomart (required for #20784)
----

> Bioconda requires reviews prior to merging pull-requests (PRs). To facilitate this, once your PR is passing tests and ready to be merged, please add the `please review & merge` label so other members of the bioconda community can have a look at your PR and either make suggestions or merge it.

* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/contributor/guidelines.html).
* [x] This PR adds a new recipe.
* [x] AFAIK, this recipe **is directly relevant to the biological sciences**
      (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

> Everyone has access to the following BiocondaBot commands, which can be given in a comment:
>
>  * `@BiocondaBot please update` will cause the BiocondaBot to merge the master branch into a PR
>  * `@BiocondaBot please add label` will add the `please review & merge` label.
>  * `@BiocondaBot please fetch artifacts` will post links to packages and docker containers built by the CI system. You can use this to test packages locally before merging.
>
> For members of the Bioconda project, the following command is also available:
>
>  * `@BiocondaBot please merge` will cause packages/containers to be uploaded and a PR merged. Someone must approve a PR first! This has the benefit of not wasting CI build time required by manually merging PRs.
>
> If you have questions, please post them in gitter or ping `@bioconda/core` in a comment (if you are not able to directly ping `@bioconda/core` then the bot will repost your comment and enable pinging).

### CODE REVIEW

**Comment by xialiu71** [comment]
@BiocondaBot please add label

File: recipes/pybiomart/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: e9eac20db921820670c646d99725b0ee279e407379e5e8c3ec7245a07425d8fe
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
```
Comment (raivivek) [inline recipes/pybiomart/meta.yaml]: Perhaps add `--ignore-installed --no-deps` since we are specifying dependencies separately in requirements below?

Comment (npavlovikj) [reply recipes/pybiomart/meta.yaml]: FYI, `{{ PYTHON }} -m pip install . -vv` actually performs `{{ PYTHON }} -m pip install --no-deps --ignore-installed .`, so no need for changes here.

**Comment by npavlovikj** [comment]
@BiocondaBot please fetch artifacts

**Comment by npavlovikj** [comment]
@BiocondaBot please merge

**Comment by npavlovikj** [comment]
@raivivek , looks like you need to approve the change you have requested before we can merge the recipe.

File: recipes/pybiomart/meta.yaml (hunk 1 above)
Comment (raivivek) [reply recipes/pybiomart/meta.yaml]: @npavlovikj Thanks for the tip! I was unaware of it.

**Comment by raivivek** [comment]
@bioconda-bot please merge

**Comment by xialiu71** [comment]
Thank you both!

**Pull request merged**